service/s3/external/external.go

```diff
@@ -0,0 +1,9 @@
+// Code generated by private/model/cli/gen-api/main.go. DO NOT EDIT.
+
+package external
+
+type WithUseARNRegion bool
+
+func (v WithUseARNRegion) GetS3UseARNRegion() (value bool, ok bool, err error) {
+	return bool(v), true, nil
+}
```

Comment:
It might be better for the SDK not to generate this file at all. On second look at a sub package with just this file seems excessive.